PalasoUIWindowsForms/Reporting/MemoryManagement.cs

```diff
@@ -37,7 +37,9 @@ public static class MemoryManagement
 		public static bool CheckMemory(bool minor, string eventDescription, bool okToBotherUser)
 		{
 			var heapMem = GC.GetTotalMemory(true); // first, as it may reduce other numbers
+			var is64BitProcess = IntPtr.Size == 8; // according to MSDN
 			long memorySize64;
```

Comment:
The 32-bit variants of the properties in the Process class are all marked obsolete.  I suppose retaining the 64 on the names merely reminds us that these are long (or ulong) variables instead of int (or uint) variables.  Since the names mimic the names of the Process properties, you might argue that they're helpful reminders that we've accessed the recommended properties.  In any case, I would argue that the 64 is correct.  Helpful?  I suppose opinions could vary.
Note that the PAE extensions to the chip architecture means that even a 32-bit OS with 32-bit processes can have more than 4GB of RAM available for use even if any one process can only directly access 2GB.